Money*/Money*/Utils/Theme.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Theme.swift
+//  Money*
+//
+//  Created by tran.duc.tan on 3/13/19.
+//  Copyright © 2019 tranductanb. All rights reserved.
+//
+
+import UIKit
+
+struct Theme {
+    struct Color {
+        static let ivory = UIColor(red: 255 / 255, green: 255 / 255, blue: 243 / 255, alpha: 1)
```

Comment:
```suggestion
        @nonobjc class var ivory: UIColor {
            return UIColor(red: 255 / 255, green: 255 / 255, blue: 243 / 255, alpha: 1)
        }
```
viết trong extention của UIColor, tương tự với những cái bên dưới
khi gọi thì có thể gọi luôn là `.ivory` thay vì `UIColor.ivory`